site/content/docs/cli/constructs.md

```diff
@@ -114,38 +114,95 @@ All resources you can create and manage using the Checkly CLI are derived from "
 You can use standard JS/TS programming to use these constructs to create the monitoring setup of your
 choice. Loops, variables, if-statements, file imports, extensions etc.
 
-## Project
+## `Project`
+
+A `Project` defines core settings and defaults for the CLI and other constructs like Checks. In many cases, you can just
+use set defaults for your Checks in the `checks` property and override them occasionally at the Check or CheckGroup level. 
+
+```ts
+import { defineConfig } from '@checkly/cli'
+
+const config = {
+  projectName: 'Website Monitoring',
+  logicalId: 'website-monitoring-1',
+  repoUrl: 'https://github.com/acme/website',
+  checks: {
+    activated: true,
+    muted: false,
+    runtimeId: '2022.10',
+    frequency: 5,
+    locations: ['us-east-1', 'eu-west-1'],
+    tags: ['website', 'api'],
+    alertChannels: [],
+    checkMatch: '**/__checks__/*.check.ts',
+    ignoreDirectoriesMatch: [],
+      browserChecks: {
+      frequency: 10,
+      testMatch: '**/__checks__/*.spec.ts',
+    },
+  },
+  cli: {
+    runLocation: 'eu-west-1',
+    privateRunLocation: 'private-dc1'
+  }
+}
+
+export default defineConfig(config)
+```
+
+- `projectName` : A friendly name for your Project.
+- `logicalId` : A unique identifier for this Project. Like all logical ID's, this should be stable.
+- `repoUrl` : An optional URL to a Git repository.
```

Comment:
Just FYI yday this wasn't optional. :)

Reply:
I will make sure it is optional ASAP in the CLI